### PR TITLE
feat(legacy-plugin-chart-markup): add controls to markup chart

### PR DESCRIPTION
**Pull request by pkdotson**
💔 Breaking Changes
   
🏆 Enhancements
     Add controls to markup chart plugin.
📜 Documentation

🐛 Bug Fix

🏠 Internal

### CODE REVIEW

File: plugins/legacy-plugin-chart-markup/src/controlPanel.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { t } from '@superset-ui/translation';
+import { validateNonEmpty } from '@superset-ui/validator';
+import { formatSelectOptions } from '@superset-ui/control-utils';
+
+console.log('what are going on with these controls')
```
Comment (kristw) [inline plugins/legacy-plugin-chart-markup/src/controlPanel.js:23]: remove `console.log`

File: plugins/legacy-plugin-chart-markup/src/controlPanel.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                    : 'markdown',
+              }),
+              default: '',
+            },
+          },
+        ],
+      ],
+    },
+  ],
+  sectionOverrides: {
```
Comment (kristw) [inline plugins/legacy-plugin-chart-markup/src/controlPanel.js:64]: Is this section needed?

Comment (rusackas) [reply plugins/legacy-plugin-chart-markup/src/controlPanel.js:64]: @kristw I think they are needed (or at least relevant). There are a few plugins that do this... iframe, markup, and separator. It appears this basically removes the `time_range` control  that's seemingly injected automatically in some cases as part of the `druidTimeSeries`. Maybe we can find a better way of addressing this in the future, but it seems out of the scope of this PR, which is just to migrate the code that's there already.

Comment (kristw) [reply plugins/legacy-plugin-chart-markup/src/controlPanel.js:63]: got it

**Pull request merged**